test/specs/lightbox.js

```diff
@@ -1,11 +1,72 @@
 /* eslint-env jasmine */
+/* global affix */
 
+import $ from 'jquery';
 import lightbox from '../../lightbox';
 
 describe('lightbox', function() {
+  const FADE_TIME = 200;
+  const BLOCKING_CLASS = '.js-blocking';
+  const LIGHTBOX_CLASS = '.js-lightbox';
+  const NEXT_CLASS = '.js-next';
+  const CLOSE_CLASS = '.js-close';
+  const imagePath = (id) => `/base/test/fixtures/images/img_${id}.png`;
+
+  // This is temporary. I don't want to update the code to add proper
+  // events before putting the existing code inside a test harness.
+  // The duration is just a bit longer than the fadeIn + fadeOut duration.
+  function tempWait(fn) {
+    setTimeout(fn, 2 * FADE_TIME + 16);
+  }
+
+  // must close it before running another test as there is no destructor
+  function close(done) {
+    tempWait(() => {
+      $(CLOSE_CLASS).click();
+      tempWait(() => {
+        expect($(BLOCKING_CLASS)).not.toBeVisible();
```

Comment:
I would also add this assertion to the top level `afterEach`, so if a future maintainer fails to call `close` in their test, they'll get a failure.